api/models/fixtures/agent_label.json

```diff
@@ -75,6 +75,42 @@
         },
         "Yes"
       ]
+    },
+    "RegionHasCustomTopicUpdate": {
```

Comment:
If you need a conditional then it should be named something that describes it better. This conditional seems to suggest that it is whether or not the region allows custom topic updates but that is not the case. This conditional describes whether or not the region allows the runtime attribute of a lambda function to be updated.